jaeger-core/src/test/java/com/uber/jaeger/TracerTagsTest.java

```diff
@@ -61,39 +61,25 @@ public TracerTagsTest(SpanType spanType, Map<String, Object> expectedTags) {
   @Parameterized.Parameters(name = "{index}: {0}")
   public static Collection<Object[]> data() {
     Tracer tracer = new Tracer.Builder("x", null, null).build();
-    String hostname = tracer.getHostName();
 
     Map<String, Object> rootTags = new HashMap<>();
-    rootTags.put("jaeger.version", tracer.getVersion());
-    rootTags.put("jaeger.hostname", hostname);
-    rootTags.put("tracer.tag.str", "y");
-    rootTags.put("tracer.tag.bool", true);
-    rootTags.put("tracer.tag.num", 1);
+    rootTags.put("jaeger.version", SENTINEL);
+    rootTags.put("jaeger.hostname", SENTINEL);
+    rootTags.put("tracer.tag.str", SENTINEL);
```

Comment:
Well, I think this whole test was meant for `zipkin.thrift`, because it had the logic of only reporting the process tags in the first-in-process span. You're essentially changing everything to SENTINEL, meaning these tags are not going to be set on the span.